lib/ansible/modules/user.py

```diff
@@ -183,6 +183,14 @@
             - If no passphrase is provided, the SSH key will default to having no passphrase.
         type: str
         version_added: "0.9"
+    ssh_key_rounds:
+        description:
+            - Optionally specify the number of KDF (key derivation function) rounds used to provide higher resistance to brute-force password cracking.
+            - If set to higher values, decrypting the encrypted private key file will take longer.
+            - It only takes effect when O(ssh_key_passphrase) is provided because in that case the private key is encrypted.
+            - Default value depends on Platform and Distribution. Standard is L(16), but newer Fedora, Ubuntu or FreeBSD use L(24) for example.
```

Comment:
I believe this is incorrect, Ubuntu 24.04 uses 16 rounds by default: https://manpages.ubuntu.com/manpages/noble/man1/ssh-keygen.1.html

Reply:
Documentation differs from actual test result. That's why i had to adapt the tests.

Reply:
TIL.

```
$ python3 -m openssh_key test_id_ed25519 --passphrase secret_passphrase | jq .kdf_options.data.rounds
24
```

Reply:
` Default value depends on Platform and Distribution.` is enough, any example will be outdated at one point and require updating.

Reply:
The same applies to the automated tests. So should i still change this? @bcoca